HW 1/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework</title>
+</head>
+<body>
+
+    <button id="myBtn"> Try it </button>
+    
+    <button class="myTogBtn"> Toggle Button </button>
+
+    <script>
+
+        let tryButton = document.getElementById("myBtn");
+        const firstLabel = "myBtn";
+        let toggButton = document.getElementById("myTogBtn");

```

Comment:
and this variable could be const.
The other thing is - you're not using it here, then why do you need it?